dts/arm/st/f4/stm32f427.dtsi

```diff
@@ -86,5 +87,21 @@
 			status = "disabled";
 			label = "SPI_6";
 		};
+
+		fmc: memory-controller@52004000 {
```

Comment:
Would you mind adding nodes as well for f437 and f446 ?
This would make the support complete for the F4 family.

Reply:
Yeah no problem. Fixed the node address as well, realized now it was not matching the reg property.